openedx/features/course_experience/tests/views/test_course_home.py

```diff
@@ -213,7 +213,7 @@ def test_queries(self):
 
         # Fetch the view and verify the query counts
         # TODO: decrease query count as part of REVO-28
-        with self.assertNumQueries(89, table_blacklist=QUERY_COUNT_TABLE_BLACKLIST):
+        with self.assertNumQueries(91, table_blacklist=QUERY_COUNT_TABLE_BLACKLIST):
```

Comment:
It's always +2 queries for the sequence as a whole right? Not on a per-item basis?

Reply:
That's right.